.travis.yml

```diff
@@ -3,7 +3,8 @@ git:
   depth: false
 cache:
   ccache: true
-  directories: $HOME/Library/Caches/Homebrew
+  directories: 
+    - "$HOME/Library/Caches/Homebrew"
```

Comment:
What is the purpose of this change?
Let's only make changes which are necessary to simplify peer-review and reduce risk of breaking something.

Reply:
It throws an error in my editor.